docs/DevGuide/CodeReviewGuide.md

```diff
@@ -15,73 +15,88 @@ Here we summarize what we view as the more important portions of the guides.
 Stylistic Items:
 
 * Adhere to [Google style](https://google.github.io/styleguide/cppguide.html).
-Can use
-[`clang-format -style=google` (here)](http://clang.llvm.org/docs/ClangFormat.html).
+  Can use [`clang-format -style=google` (here)]
```

Comment:
Either remove the `(here)`, or reduce the clickable text to only `here`.

Reply:
The code does not highlight (see the current documentation), though I don't see why it shouldn't be clickable. Care to provide a justification?

Reply:
I think what @fmahebert is trying to say is to either remove " (here)" from the bracketed text or only put "(here)" in the brackets as everything in the [] is clickable.  I prefer removing the " (here)" as no other clickable links on the page have it.

Reply:
My thinking was: it is common (and therefore expected) for the clickable text to be the same as the highlighted text. When I noticed here that the clickable text is much larger than the highlighted text, I was surprised.

Your points are good, and I see that my expected behavior is difficult to achieve here. An alternative that would surprise me less would be to make the whole sentence clickable:
```[Can use `clang-format -style=google`](link)```

But I also won't be offended if it stays as is ;)

Reply:
Yes, but the issue is that the code does not appear to be a link. See [here](https://sxs-collaboration.github.io/spectre/code_review_guide.html). Maybe the answer is to actually have "Can use `code`" all in the brackets, removing the here? That way the whole sentence is a link, where the "Can use" is highlighted and therefore obviously clickable.

Reply:
That is in fact what I was suggesting in my comment :P

Reply:
Woops, I hadn't seen @fmahebert's reply. I think `(link)` is a good suggestion and I'd be happy with that :) @kidder ?

Reply:
Whoops on my part as well, @kidder commented while I was editing and I never saw his post until I reloaded the page right now.

@nilsdeppe, if you mean replacing the appended `(here)` with an appended `(link)`, that would already be an improvement over the current state. I still slightly prefer the whole-sentence approach.

Reply:
Ah now I see what you are saying; I like @fmahebert 's whole-sentence suggestion. (And not having anything visible in ())